sdk/core/Azure.Core/tests/TestFramework/LiveOnlyAttribute.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+
+namespace Azure.Core.Testing
+{
+    /// <summary>
+    /// Attribute on test assemblies, classes, or methods that run only against live resources.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = true, Inherited = true)]
```

Comment:
AllowMultiple?

Reply:
Originally it wasn't, but since samples are often split across partial classes - both of which were attributed with `[Category("Live")]` - I did this to be able to attribute both classes as well. It shouldn't hurt anything, though I'm totally willing to disallow multiple and remove it from one class (probably the sync classes, since async is better to use/recommend).